_layouts/repos.html

```diff
@@ -90,7 +90,7 @@
                     <td>
                       {% for p in r.packages %}
                         {% assign labeled_package = p[1] %}
-                        <a href="{{site.baseurl}}/p/{{p[0]}}" class="label label-primary pkg-label">{% include peer_label.html %}</a>
```

Comment:
Actually I believe that repos.html is now unused, it should have been removed in #522  So you can just delete this file instead of modifying it if you want.

Reply:
Thanks for catching that, clearing it out is great to do.